uaclient/cli.py

```diff
@@ -1372,22 +1377,23 @@ def action_api(args, *, cfg):
 
 
 @assert_root
-@assert_lock_file("pro auto-attach")
-def action_auto_attach(args, *, cfg):
-    if cfg.is_attached:
-        raise exceptions.AlreadyAttachedOnPROError()
-
-    skip_auto_attach = actions.should_disable_auto_attach(cfg)
```

Comment:
This is arguably out of scope so feel free to say no.

There is only one other caller of `actions.should_disable_auto_attach(cfg)` in lib/auto_attach.py and it can be replaced by catching `AutoAttachDisabledError`. If we do that as well then we can remove the `actions.should_disable_auto_attach` function. Then we can move the log statement defined in that function here.

Reply:
Let do that!

Reply:
Done.